.evergreen/generate_evergreen_tasks.js

```diff
@@ -105,7 +105,9 @@ BASE_TASKS.push({
       TOPOLOGY: 'server',
       REQUIRE_API_VERSION: '1',
       MONGODB_API_VERSION: '1',
-      AUTH: 'auth'
+      AUTH: 'auth',
+      TEST_CSFLE: 'true',
```

Comment:
We actually have three different variables we use to control whether or not we run FLE tests:

1. CLIENT_ENCRYPTION
2. TEST_CSFLE
3. TEST_SOCKS5_CSFLE

Someday it would be nice to consolidate on one.  I can make that change here if we'd like but chose not to because this PR is large.

As for the changes you do see in this PR with these variables - I made `CLIENT_ENCRYPTION` required in the run tests evergreen function and had to add it in a number of places.  I added TEST_CSFLE too just for consistency.